src/Microsoft.DocAsCode.Build.Common/RedirectionFileHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Common
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    public class RedirectionFileHelper

```

Comment:
This is not a correct implementation. As we discussed, please talk to validation library and see whether they can embed this logic.